src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <ul class="page__calendar calendar">
+      <li class="calendar__day calendar__day--1"></li>
```

Comment:
```suggestion
      <li class="calendar__day"></li>
```
Используй `nth-child`

Reply:
Исправил